.github/workflows/generate-clients.yml

```diff
@@ -5,6 +5,7 @@ on:
       - main
     paths:
       - dj/**
+  workflow_dispatch:
```

Comment:
I added this so that we can also manually trigger the workflow to generate the client. I'll do that after this PR is merged so that it can regenerate with the new name `djopenapi`.